app/components/ui/GDocEmbed/GDocEmbed.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Iframe from 'react-iframe';
+import Footer from '../Footer/Footer';
+
+import styles from './GDocEmbed.scss';
+
+
+export default class GDocEmbed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const { pageHeader } = this.props;
+    const { GDocURL } = this.props;
+    return (
+      <React.Fragment>
+        <article className={styles.GPage}>
+          <header className={styles.GHeader}>
+            <h1>
+              {pageHeader}
+            </h1>
+            <p>
+              This list is compiled to link individuals experiencing homelessness to resources.
+              We will be updating the list continuously, when services change.
+              <br />
+              Please email
+              <a href="mailto:contact@sheltertech.org">
+                {' '}
```

Comment:
I'd put the `{' '}` above and outside the `<a>` tag, since it's kind of weird to have the space character before a link to be clickable.